python/psautohint/autohint.py

```diff
@@ -282,10 +282,9 @@ def hintFiles(options):
     if options.reference_font:
         hintFile(options, options.reference_font, None, reference_master=True)
     for i, path in enumerate(options.inputPaths):
-        try:
+        outpath = None
+        if options.outputPaths is not None and i < len(options.outputPaths):
```

Comment:
maybe you could add be a specific test with multiple -o options when there are multiple input_paths/output_paths
https://github.com/adobe-type-tools/psautohint/blob/6293d470ed157e018f4a067407e888977619f17e/python/psautohint/__main__.py#L489